pkgs/development/python-modules/nassl/default.nix

```diff
@@ -108,6 +114,10 @@ buildPythonPackage rec {
     invoke package.wheel
   '';
 
+  doCheck = true;
+
+  pythonImportsCheck = [ "nassl" ];
+
   checkInputs = [ pytestCheckHook ];
```

Comment:
This doesn't make sense combined with `checkPhase`. Use `pytestFlagsArray`.

Reply:
Adapted.